packages/wonder-blocks-dropdown/components/dropdown-launcher.js

```diff
@@ -207,6 +212,45 @@ export default class DropdownLauncher extends React.Component<Props, State> {
             </span>
         );
 
+        /*         TODO: Enforce better type check here
+        Currently this only looks for the 'icon' prop to decide
+        if an element is a IconButton or not as the 'isClassOf' method isnt present
+        */
+        if (childAnchor.props.icon) {
+            opener = (
+                <span
+                    ref={this.handleOpenerRef}
+                    disabled={!dropdownItems.length || disabled}
+                >
+                    {children(this._openDropdown)}
+                </span>
+            );
+        } else {
+            opener = (
+                <View
```

Comment:
This should be moved into the example.  Also, we want to be able to provide information about the state of the opener to the `children` callback, i.e. the pressed, focused, and hovered state that `ClickableBehavior`.  This prototype need to integrate `ClickableBehavior` into `DropdownLauncher` some how.